cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -2,21 +2,24 @@
 
 {% macro render(value, settings, form_id) %}
 
-    {% set form = settings.form %}
-
-    {# JS-disabled result handling #}
-
-<div class="o-feedback
```

Comment:
The code in `feedback-form.js` looks for `o-feedback` as its base class. I'll add it in another PR since this is already merged